src/pybind/mgr/nfs/export.py

```diff
@@ -651,10 +655,35 @@ def _create_user_key(self, cluster_id: str, entity: str, path: str, fs_name: str
         log.info(f"Export user created is {json_res[0]['entity']}")
         return json_res[0]['key']
 
+    def _check_earmark(self, earmark_resolver: CephFSEarmarkResolver, path: str,
+                       fs_name: str) -> None:
+        if earmark_resolver:
```

Comment:
just curious, given `earmark_resolver` is a required arg to this function, is there actually any chance it could resolve to `False`? Is there some state the  `CephFSEarmarkResolver` class can be in that causes that?

Reply:
I think probably not, since we ensure that CephFSEarmarkResolver(self) is passed directly from the module itself. This should guarantee that earmark_resolver is always initialized properly. I just followed this similar way for smb module

Reply:
I guess i can drop `if earmark_resolver:`